src/Robo/Commands/Tests/DrupalTestCommand.php

```diff
@@ -106,17 +106,28 @@ public function run() {
    */
   public function executeTests() {
     $this->logger->notice("Ensure that you have installed Drupal Core's dev dependencies (such as via https://github.com/webflo/drupal-core-require-dev) prior to running this command.");
-    if (empty(shell_exec('which java'))) {
-      $this->logger->warning("Cannot find Java. Drupal tests require Selenium, which requires Java. Drupal tests will be skipped.");
-      return;
-    }
     if ($this->drupalTestRunner == 'phpunit') {
+      if (empty(shell_exec('which java'))) {
```

Comment:
Shouldn't this be a different method to avoid code duplication?

Reply:
This would be perfect 🥑 in ruby.

`return or test_java`

Reply:
Yeah, I was hoping to avoid any refactoring but I'll move it to a new method. This whole class needs to be cleaned up at some point to have a prerequisite check.

Reply:
Got it!